src/dotty/tools/dotc/transform/Erasure.scala

```diff
@@ -263,6 +263,12 @@ object Erasure extends TypeTestsCasts{
           else
             cast(tree, pt)
       }
+
+
+    /** Is `tpe` a type which is a bottom type for Dotty, but not a bottom type for JVM */
```

Comment:
There is no bottom type on the JVM so I don't really understand the naming of this function, how is it different from `Definitions#isBottomType`?